src/Components/Blazor/WebAssembly.Authentication/src/Services/AccessTokenRequestOptions.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Components.WebAssembly.Authentication
+{
+    /// <summary>
+    /// Represents the options for provisioning an access token on behalf of a user.
+    /// </summary>
+    public class AccessTokenRequestOptions
+    {
+        /// <summary>
+        /// Gets or sets the list of scopes to request for the token.
+        /// </summary>
+        public IReadOnlyList<string> Scopes { get; set; }
```

Comment:
Is there any use case for allowing this to be `null`? If not, should it be a required-non-null constructor parameter?